source/_components/google_assistant.markdown

```diff
@@ -138,7 +138,7 @@ api_key:
   description: Your Homegraph API key (for the `google_assistant.request_sync` service)
   required: false
   type: string
-expose_by_default:
+expose_by_default: (Changed in 0.84. Will not sync devices if set to true)
```

Comment:
There is no such change

Reply:
Alright, I withdraw my previous review. In 0.84, there was a change to fix a bug exists in 0.80-0.83.

This sentence is not right anyway, true/false is reversed and the actual situation is more complex. Moreover, since current document technically correct, I don't think we need add a note for a changed made in half year ago.

Reply:
If you set expose_by_default to *false*, and expose specific domains nothing will sync. It seems like you need to expose specific entities. This can be very confusing when setting stuff up, no reason for expose_by_default to even exist.